RecoTracker/MkFitCore/interface/Config.h

```diff
@@ -91,6 +91,15 @@ namespace mkfit {
     constexpr float mag_b1 = 7.53701e-06;
     constexpr float mag_a = 2.43878e-11;
 
+    // Config for "fake" hit addition
+    // Only if fake_hit_idx==-1, then count as missing hit for candidate score
+    constexpr int hit_miss_idx_ = -1;        //  hit is missed
```

Comment:
cms style for constants is to have a `k` prefix and also use camelCap

also, the publicly available constants do not need to have a trailing `_`
so, e.g. `kHitMissedIdx`

It's not obvious to me that `Config.h` is the right place. Perhaps `Hit` class is a better place to have it?

Reply:
Should be done in commit [9211e52](https://github.com/trackreco/cmssw/pull/90/commits/9211e52277a1a5d98b82abde55618fbea81bdae3).

Reply:
(both moving to `Hit` class and using a more appropriate naming convention)